runelite-client/src/test/java/net/runelite/client/plugins/screenshot/ScreenshotPluginTest.java

```diff
@@ -113,8 +111,14 @@ public void testClueScroll()
 		ChatMessage chatMessageEvent = new ChatMessage(null, GAMEMESSAGE, "Seth", CLUE_SCROLL, null, 0);
 		screenshotPlugin.onChatMessage(chatMessageEvent);
 
-		assertEquals("medium", screenshotPlugin.getClueType());
-		assertEquals(28, screenshotPlugin.getClueNumber());
+		WidgetLoaded event = new WidgetLoaded();
+		event.setGroupId(CLUE_SCROLL_REWARD_GROUP_ID);
+		screenshotPlugin.onWidgetLoaded(event);
+
+
+		//assertEquals("medium", screenshotPlugin.getClueType());
```

Comment:
Why are you commenting out old tests? This is not the right way to change the test suite for this plugin.

Reply:
I commented it out to test if my implementation was correct. Didn't see the need to create a more tests when I could just check my value from there. I initially didn't comment it out but didn't know if having two assertEquals would mess up the check.

Reply:
You can have any number of JUnit assertions in a single test case. The only important thing is that none of them fail.

Reply:
Ok. Thanks for clarifying.